backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt

```diff
@@ -3,28 +3,31 @@ package app.ehrenamtskarte.backend
 import app.ehrenamtskarte.backend.common.database.Database
 import app.ehrenamtskarte.backend.common.webservice.GraphQLHandler
 import app.ehrenamtskarte.backend.common.webservice.WebService
+import app.ehrenamtskarte.backend.config.BackendConfiguration
 import app.ehrenamtskarte.backend.stores.importer.LbeDataImporter
 import com.expediagroup.graphql.extensions.print
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.core.ProgramResult
+import com.github.ajalt.clikt.core.requireObject
 import com.github.ajalt.clikt.core.subcommands
 import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
 import java.io.File
 
 class Entry : CliktCommand() {
-    companion object {
-        fun isProductionEnvironment(): Boolean {
-            return System.getProperty("app.production", "").isNotEmpty()
-        }
-    }
+    private val config by option().file(canBeDir = false, mustBeReadable = true).required()
```

Comment:
This should be optional. Some search for a config should be performed here. First look in the current working directory. If there is no `config.yml` then look in `/etc/entitlementcard.yml/` and if no config if found we could warn and choose a hardcoded default.

Reply:
Done (without the warning part)